<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="0" failed="0" skipped="4">
    <reporter-output>
    </reporter-output>
    <suite started-at="2018-07-08T11:43:52Z" name="Suite" finished-at="2018-07-08T11:43:52Z" duration-ms="330">
        <groups>
        </groups>
        <test started-at="2018-07-08T11:43:52Z" name="Validating User login" finished-at="2018-07-08T11:43:52Z"
              duration-ms="130">
            <class name="Academy.HomePage">
                <test-method is-config="true" signature="initialize()[pri:0, instance:Academy.HomePage@4b3c354a]"
                             started-at="2018-07-08T11:43:52Z" name="initialize" finished-at="2018-07-08T11:43:52Z"
                             duration-ms="21" status="FAIL">
                    <exception class="java.io.FileNotFoundException">
                        <message>
                            <![CDATA[C:\Users\rahul\E2EProject\src\main\java\resources\data.properties (The system cannot find the path specified)]]>
                        </message>
                        <full-stacktrace>
                            <![CDATA[java.io.FileNotFoundException: C:\Users\rahul\E2EProject\src\main\java\resources\data.properties (The system cannot find the path specified)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(Unknown Source)
at java.base/java.io.FileInputStream.<init>(Unknown Source)
at java.base/java.io.FileInputStream.<init>(Unknown Source)
at resources.base.initializeDriver(base.java:26)
at Academy.HomePage.initialize(HomePage.java:25)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
                        </full-stacktrace>
                    </exception> <!-- java.io.FileNotFoundException -->
                    <reporter-output>
                    </reporter-output>
                </test-method> <!-- initialize -->
                <test-method
                        signature="basePageNavigation(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:Academy.HomePage@4b3c354a]"
                        started-at="2018-07-08T11:43:52Z" name="basePageNavigation" data-provider="getData"
                        finished-at="2018-07-08T11:43:52Z" duration-ms="0" status="SKIP">
                    <exception class="java.io.FileNotFoundException">
                        <message>
                            <![CDATA[C:\Users\rahul\E2EProject\src\main\java\resources\data.properties (The system cannot find the path specified)]]>
                        </message>
                        <full-stacktrace>
                            <![CDATA[java.io.FileNotFoundException: C:\Users\rahul\E2EProject\src\main\java\resources\data.properties (The system cannot find the path specified)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(Unknown Source)
at java.base/java.io.FileInputStream.<init>(Unknown Source)
at java.base/java.io.FileInputStream.<init>(Unknown Source)
at resources.base.initializeDriver(base.java:26)
at Academy.HomePage.initialize(HomePage.java:25)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
                        </full-stacktrace>
                    </exception> <!-- java.io.FileNotFoundException -->
                    <reporter-output>
                    </reporter-output>
                </test-method> <!-- basePageNavigation -->
                <test-method
                        signature="basePageNavigation(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:Academy.HomePage@4b3c354a]"
                        started-at="2018-07-08T11:43:52Z" name="basePageNavigation" data-provider="getData"
                        finished-at="2018-07-08T11:43:52Z" duration-ms="0" status="SKIP">
                    <exception class="java.io.FileNotFoundException">
                        <message>
                            <![CDATA[C:\Users\rahul\E2EProject\src\main\java\resources\data.properties (The system cannot find the path specified)]]>
                        </message>
                        <full-stacktrace>
                            <![CDATA[java.io.FileNotFoundException: C:\Users\rahul\E2EProject\src\main\java\resources\data.properties (The system cannot find the path specified)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(Unknown Source)
at java.base/java.io.FileInputStream.<init>(Unknown Source)
at java.base/java.io.FileInputStream.<init>(Unknown Source)
at resources.base.initializeDriver(base.java:26)
at Academy.HomePage.initialize(HomePage.java:25)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
                        </full-stacktrace>
                    </exception> <!-- java.io.FileNotFoundException -->
                    <reporter-output>
                    </reporter-output>
                </test-method> <!-- basePageNavigation -->
                <test-method is-config="true" signature="teardown()[pri:0, instance:Academy.HomePage@4b3c354a]"
                             started-at="2018-07-08T11:43:52Z" name="teardown" finished-at="2018-07-08T11:43:52Z"
                             duration-ms="0" status="SKIP">
                    <reporter-output>
                    </reporter-output>
                </test-method> <!-- teardown -->
            </class> <!-- Academy.HomePage -->
        </test> <!-- Validating User login -->
        <test started-at="2018-07-08T11:43:52Z" name="Validating title in the Page" finished-at="2018-07-08T11:43:52Z"
              duration-ms="17">
            <class name="Academy.validateTitle">
                <test-method is-config="true" signature="initialize()[pri:0, instance:Academy.validateTitle@7d3430a7]"
                             started-at="2018-07-08T11:43:52Z" name="initialize" finished-at="2018-07-08T11:43:52Z"
                             duration-ms="1" status="FAIL">
                    <exception class="java.io.FileNotFoundException">
                        <message>
                            <![CDATA[C:\Users\rahul\E2EProject\src\main\java\resources\data.properties (The system cannot find the path specified)]]>
                        </message>
                        <full-stacktrace>
                            <![CDATA[java.io.FileNotFoundException: C:\Users\rahul\E2EProject\src\main\java\resources\data.properties (The system cannot find the path specified)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(Unknown Source)
at java.base/java.io.FileInputStream.<init>(Unknown Source)
at java.base/java.io.FileInputStream.<init>(Unknown Source)
at resources.base.initializeDriver(base.java:26)
at Academy.validateTitle.initialize(validateTitle.java:26)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
                        </full-stacktrace>
                    </exception> <!-- java.io.FileNotFoundException -->
                    <reporter-output>
                    </reporter-output>
                </test-method> <!-- initialize -->
                <test-method signature="validateAppTitle()[pri:0, instance:Academy.validateTitle@7d3430a7]"
                             started-at="2018-07-08T11:43:52Z" name="validateAppTitle"
                             finished-at="2018-07-08T11:43:52Z" duration-ms="0" status="SKIP">
                    <exception class="java.io.FileNotFoundException">
                        <message>
                            <![CDATA[C:\Users\rahul\E2EProject\src\main\java\resources\data.properties (The system cannot find the path specified)]]>
                        </message>
                        <full-stacktrace>
                            <![CDATA[java.io.FileNotFoundException: C:\Users\rahul\E2EProject\src\main\java\resources\data.properties (The system cannot find the path specified)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(Unknown Source)
at java.base/java.io.FileInputStream.<init>(Unknown Source)
at java.base/java.io.FileInputStream.<init>(Unknown Source)
at resources.base.initializeDriver(base.java:26)
at Academy.validateTitle.initialize(validateTitle.java:26)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
                        </full-stacktrace>
                    </exception> <!-- java.io.FileNotFoundException -->
                    <reporter-output>
                    </reporter-output>
                </test-method> <!-- validateAppTitle -->
                <test-method is-config="true" signature="teardown()[pri:0, instance:Academy.validateTitle@7d3430a7]"
                             started-at="2018-07-08T11:43:52Z" name="teardown" finished-at="2018-07-08T11:43:52Z"
                             duration-ms="0" status="SKIP">
                    <reporter-output>
                    </reporter-output>
                </test-method> <!-- teardown -->
            </class> <!-- Academy.validateTitle -->
        </test> <!-- Validating title in the Page -->
        <test started-at="2018-07-08T11:43:52Z" name="App Testing" finished-at="2018-07-08T11:43:52Z" duration-ms="1">
        </test> <!-- App Testing -->
        <test started-at="2018-07-08T11:43:52Z" name="Validating navigation Bar" finished-at="2018-07-08T11:43:52Z"
              duration-ms="17">
            <class name="Academy.validateNavigationBar">
                <test-method is-config="true"
                             signature="initialize()[pri:0, instance:Academy.validateNavigationBar@c1fca1e]"
                             started-at="2018-07-08T11:43:52Z" name="initialize" finished-at="2018-07-08T11:43:52Z"
                             duration-ms="1" status="FAIL">
                    <exception class="java.io.FileNotFoundException">
                        <message>
                            <![CDATA[C:\Users\rahul\E2EProject\src\main\java\resources\data.properties (The system cannot find the path specified)]]>
                        </message>
                        <full-stacktrace>
                            <![CDATA[java.io.FileNotFoundException: C:\Users\rahul\E2EProject\src\main\java\resources\data.properties (The system cannot find the path specified)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(Unknown Source)
at java.base/java.io.FileInputStream.<init>(Unknown Source)
at java.base/java.io.FileInputStream.<init>(Unknown Source)
at resources.base.initializeDriver(base.java:26)
at Academy.validateNavigationBar.initialize(validateNavigationBar.java:30)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
                        </full-stacktrace>
                    </exception> <!-- java.io.FileNotFoundException -->
                    <reporter-output>
                    </reporter-output>
                </test-method> <!-- initialize -->
                <test-method signature="validateAppNavBar()[pri:0, instance:Academy.validateNavigationBar@c1fca1e]"
                             started-at="2018-07-08T11:43:52Z" name="validateAppNavBar"
                             finished-at="2018-07-08T11:43:52Z" duration-ms="0" status="SKIP">
                    <exception class="java.io.FileNotFoundException">
                        <message>
                            <![CDATA[C:\Users\rahul\E2EProject\src\main\java\resources\data.properties (The system cannot find the path specified)]]>
                        </message>
                        <full-stacktrace>
                            <![CDATA[java.io.FileNotFoundException: C:\Users\rahul\E2EProject\src\main\java\resources\data.properties (The system cannot find the path specified)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(Unknown Source)
at java.base/java.io.FileInputStream.<init>(Unknown Source)
at java.base/java.io.FileInputStream.<init>(Unknown Source)
at resources.base.initializeDriver(base.java:26)
at Academy.validateNavigationBar.initialize(validateNavigationBar.java:30)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
                        </full-stacktrace>
                    </exception> <!-- java.io.FileNotFoundException -->
                    <reporter-output>
                    </reporter-output>
                </test-method> <!-- validateAppNavBar -->
                <test-method is-config="true"
                             signature="teardown()[pri:0, instance:Academy.validateNavigationBar@c1fca1e]"
                             started-at="2018-07-08T11:43:52Z" name="teardown" finished-at="2018-07-08T11:43:52Z"
                             duration-ms="0" status="SKIP">
                    <reporter-output>
                    </reporter-output>
                </test-method> <!-- teardown -->
            </class> <!-- Academy.validateNavigationBar -->
        </test> <!-- Validating navigation Bar -->
    </suite> <!-- Suite -->
</testng-results>
